packages/flutter/lib/src/widgets/view.dart

```diff
@@ -106,13 +145,669 @@ class View extends StatelessWidget {
     }());
     return result!;
   }
+
+  @override
+  Widget build(BuildContext context) {
+    return RawView._deprecated(
+      view: view,
+      deprecatedPipelineOwner: _deprecatedPipelineOwner,
+      deprecatedRenderView: _deprecatedRenderView,
+      builder: (BuildContext context, PipelineOwner owner) {
+        return _ViewScope(
+          view: view,
+          child: ViewHooksScope(
+            hooks: ViewHooks.of(context).copyWith(pipelineOwner: owner),
+            child: MediaQuery.fromView(
+              view: view,
+              child: child,
+            ),
+          ),
+        );
+      }
+    );
+  }
+}
+
+/// A builder for the content [Widget] of a [RawView].
+///
+/// The widget returned by the builder defines the content that is drawn into
+/// the [FlutterView] configured on the [RawView].
+///
+/// The builder is given the [PipelineOwner] that the [RawView] uses to manage
+/// its render tree. Typical builder implementations make that pipeline owner
+/// available as an attachment point for potential child views by inserting
+/// updated [ViewHooks] into the widget tree.
+///
+/// Used by [RawView.builder].
+typedef RawViewContentBuilder = Widget Function(BuildContext context, PipelineOwner owner);
+
+/// The workhorse behind the [View] widget that actually bootstraps a render
+/// tree.
+///
+/// It instantiates the [RenderView] as the root of that render tree and adds it
+/// to the [RenderViewManager] obtained from the surrounding [ViewHooks] via
+/// [ViewHooks.of] (typically, that is the [RendererBinding]). It also owns the
+/// [PipelineOwner] that manages this render tree and adds it as a child to the
+/// surrounding [ViewHooks.pipelineOwner]. This ensures that the render tree
+/// bootstrapped by this widget participates properly in frame production and
+/// hit testing.
+///
+/// The [RawView] widget faces the same limitations in terms of where it can
+/// appear in the widget tree as the [View] widget, where those limitations are
+/// documented.
```

Comment:
Reading that sentence makes me want to ask "What about where they aren't documented?"  I think that last clause in  the sentence is confusing, and could probably just be removed or be replaced with "See the [View] widget for more details." or just use a macro and include the information here.